File: EYImageSliderDemo-OC/EYImageSliderDemo-OC/EYImageSliderProgramatic.m
//
//  EYImageSliderProgramatic.m
//  EYImageSliderDemo-OC
//
//  Created by Enix Yu on 29/10/2016.
//  Copyright © 2016 RobotBros. All rights reserved.
//

#import "EYImageSliderProgramatic.h"



@interface EYImageSliderProgramatic ()

@property (nonatomic, strong) EYImageSlider *imageSlider;

@end

@implementation EYImageSliderProgramatic

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    // Create an image slider with the same size as the back view
    CGRect rect = CGRectMake(0, 0, self.view.frame.size.width, self.view.frame.size.height - 44);
    _imageSlider = [[EYImageSlider alloc] initWithFrame:rect];
    
    // Set data source, and delegate
    [self.imageSlider setDataSource:self];
    [self.imageSlider setDelegate:self];
    
    // Set text color
    [self.imageSlider setCaptionTextColor:[UIColor whiteColor]];
    
    [self.view addSubview:self.imageSlider];
}

@end

File: EYImageSliderDemo/EYImageSliderDemo/EYImageSliderProgramatic.swift
//
//  EYImageSliderProgramatic.swift
//  EYImageSliderDemo
//
//  Created by Enix Yu on 29/10/2016.
//  Copyright © 2016 RobotBros. All rights reserved.
//

import UIKit
import EYImageSlider


class EYImageSliderProgramatic: EYImageSliderBase {
    
    var imageSlider: EYImageSlider!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        imageSlider = EYImageSlider()
        imageSlider.frame = CGRect(x: 0, y: 0, width: view.frame.size.width, height: view.frame.size.height - 44)
        
        imageSlider.dataSource = self
        imageSlider.delegate = self
        
        imageSlider.captionTextColor = UIColor.white
        
        view.addSubview(imageSlider)
    }
}
